STYLE.md

```diff
@@ -0,0 +1,31 @@
+# Style Guide
+
+To ensure the consistency of a comprehensive documentation like grammY’s, there must also be a set of rules for humans to follow.
+This documents those rules.
```

Comment:
```suggestion
To ensure the consistency of a comprehensive documentation like grammY’s, there must also be a set of rules for humans to follow.
This documents those rules.

If you contribute to grammY's documentation, you do not have to follow these rules.
We are going to fix up your contributions before merging them.
That way, you can focus on writing great content, yet we end up with great style, too.

Naturally, we are always happy if we see contributions that follow this style guide upfront.
Just be aware that we might do lots of nitpicking otherwise.
```